control/vehicle_cmd_gate/src/vehicle_cmd_gate.cpp

```diff
@@ -398,6 +401,11 @@ void VehicleCmdGate::publishControlCommands(const Commands & commands)
     filtered_commands.control.longitudinal.acceleration = stop_hold_acceleration_;
   }
 
+  if (moderate_stop_interface_->is_stop_requested()) {  // if stop requested, stop the vehicle
```

Comment:
@brkay54 Could you please move this filter before the engage and emergency filters? It may be overwritten with a moderate deceleration.

Reply:
Yes you are right, thank you! I want to ask you that also should we move the pause_interface's filter upside of the emergency?

Reply:
Thank you. The pause_interface's should be the current position and has the same deceleration as engage. But it's confusing. I'll create a PR to add comments later.

Reply:
> I'll create a PR to add comments later.

FYI: Add comment in this PR https://github.com/autowarefoundation/autoware.universe/pull/3794.

Reply:
Updated the PR, thank you for your review! 🙏🏼